azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -317,6 +317,12 @@ public void execute() throws MojoExecutionException {
             // into endless loop when close, we need to call it in main thread.
             // Refer here for detail codes: https://github.com/Microsoft/ApplicationInsights-Java/blob/master/core/src
             // /main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43.java#L103
+            try {
+                // Sleep to wait ai sdk flush teletetries
+                Thread.sleep(2 * 1000);

```

Comment:
where is 2 * 1000 from? experience? evidence?

Reply:
Set FlushIntervalInSeconds(times to flush telemetries) to 1s and wait double time